static/meetings/edc-weekly-extended.ics

```diff
@@ -19,1488 +19,328 @@ TZOFFSETTO:+0200
 END:DAYLIGHT
 END:VTIMEZONE
 BEGIN:VEVENT
+ATTENDEE;CN="Westphal, Benjamin BW. (415)";RSVP=FALSE:mailto:benjamin.bw.we
```

Comment:
Currently, the attendees are also included in the calendar file. As I remember from last PRs... we discussed deleting them.

Reply:
Good point, I have removed the attendees from the file